pycatchmod/__main__.py

```diff
@@ -28,12 +28,24 @@ def compare(ctx, filename, plot):
     from pycatchmod.io.excel import compare
     compare(filename, plot)
 
+def date_parser(x):
+    import pandas
+    # try creating a date range based on the first date and the length
+    dates = pandas.period_range(x[0], periods=len(x), freq="D")
```

Comment:
That's not how period_range works. I'm doing it like this to avoid the slowness of having to parse all of the dates (it's very slow) and also issues with british/american dates.